docs/Documentation/Features/Conversations.md

```diff
@@ -175,6 +175,8 @@ and the way the player interacts with them.
         <video controls loop src="../../../_media/content/Documentation/Conversations/ChestIO.mp4" width="100%">
           Sorry, your browser doesn't support embedded videos.
         </video>
+    === "Combined Style"
+        The same as the chest style but the conversation is also displayed in the chat.
```

Comment:
If this is now the only point where this IO is documented, where can i see, what i need to write into the config for this style?

Reply:
Improved in 137f181abb124575643d34c8fea01f6086604928